schema/Mutation.graphql

```diff
@@ -84,4 +84,9 @@ type Mutation {
     Revoke user session
     """
     revokeUserSession(id: ObjectID!, displayNotice: Boolean): Boolean!
+
+    """
+    Create a new life
+    """
+    createLife(title: String!, firstname: String!, lastname: String!, description: String!, birthday: String!, hobbies: [String!]): Life!
```

Comment:
to make your life easier, you can create a GraphQL type to group all those fields as one